kong/db/declarative/init.lua

```diff
@@ -587,6 +590,8 @@ function declarative.load_into_cache(entities, meta, hash, shadow)
   local transform = meta._transform == nil and true or meta._transform
 
   for entity_name, items in pairs(entities) do
+    yield()
```

Comment:
```suggestion
    yield(true)
```

Reply:
Correct me if I'm wrong, but `for entity_name, items in pairs(entities)` only iterates known entities by entity name, it does not represents individual entity objects.

Reply:
Same idea for the following ones, they all have small loops.